source/agora/consensus/data/Transaction.d

```diff
@@ -53,6 +53,11 @@ public struct Transaction
     /// The list of newly created outputs to put in the UTXO
     public Output[] outputs;
 
+    /// This transaction may only be included in a block with `height >= unlock_height`.
+    /// Note that another tx with a lower lock time could double-spend this tx.
+    /// Using `uint` instead of `Height` as it's enough to cover 80k years.
+    public uint unlock_height = 0;
```

Comment:
Yeah, here it is. um.. uint

Reply:
> Yeah, here it is. um.. uint

There is a comment above, please read it.